src/SpecFlow.NetCore/Fixer.cs

```diff
@@ -43,14 +44,27 @@ public Fixer()
 		public void Fix(DirectoryInfo directory)
 		{
 			WriteLine("Current directory: " + directory);
-			var xproj = GetXproj(directory);
+		    featureFiles = directory.GetFiles("*.feature", SearchOption.AllDirectories);
+		    var generatedFileExists = featureFiles.Where(f => !File.Exists(f.FullName + ".cs")).ToList();
+
+            var xproj = GetXproj(directory);
```

Comment:
Definitely need to reach an agreement with @stajs regarding spacing. :smile:   Personally I use (and recommend) Keep Tabs in *.cs